composer.json

```diff
@@ -38,6 +38,7 @@
         "doctrine/orm": "^2.5",
         "jms/serializer-bundle": "^2.3",
         "ocramius/proxy-manager": "*",
+        "phpunit/phpunit": "^5.7.21 || ^6.4 || ^7.0",
```

Comment:
@alcaeus I think we should remove `|| ^6.4 || ^7.0",`, because I don't want people right tests locally using phpunit 6 or 7.